plugins/aja/CMakeLists.txt

```
@@ -14,53 +14,70 @@ find_package(LibAJANTV2 REQUIRED)
 add_library(aja MODULE)
 add_library(OBS::aja ALIAS aja)
 
+add_library(aja-support OBJECT)
+add_library(OBS::aja-support ALIAS aja-support)
```

I know this is from #9769.

> That might require explicitly setting the POSITION_INDEPENDENT_CODE target property - I just have to find out for which target specifically.

Yep, `aja-support` was guilty.
```suggestion
add_library(OBS::aja-support ALIAS aja-support)

set_target_properties(aja-support PROPERTIES POSITION_INDEPENDENT_CODE True)
```